src/control/cmd/daos_agent/mgmt_rpc.go

```diff
@@ -35,16 +35,13 @@ import (
 // Management Service proxy, handling dRPCs sent by libdaos by forwarding them
 // to MS.
 type mgmtModule struct {
-	attachInfoMutex sync.RWMutex
-	fabricMutex     sync.RWMutex
-
 	log            logging.Logger
 	sys            string
 	ctlInvoker     control.Invoker
 	cache          *InfoCache
 	monitor        *procMon
 	cliMetricsSrc  *promexp.ClientSource
-	useDefaultNUMA bool
+	useDefaultNUMA atm.Bool
```

Comment:
why change this to atomic?

Reply:
I noticed this while working on the feature. As a standard bool, it's racy. One goroutine could flip it to true while others are reading it. I think you'd have to get slammed with a bunch of requests right at agent startup to see it.